packages/lib/passport-interface/src/EncryptedStorage.ts

```diff
@@ -4,7 +4,9 @@ import {
   PCDCollection
 } from "@pcd/pcd-collection";
 import { PCDPackage, SerializedPCD } from "@pcd/pcd-types";
+import * as base64 from "base64-js";
 import stringify from "fast-json-stable-stringify";
+import pako from "pako";
```

Comment:
this looks like a pretty big package for our bundle (>1MB) -- any way we could dynamically load via bundle splitting or perhaps use a smaller package?

https://npm-compare.com/fflate,pako

Reply:
It's already a dependency for `passport-ui`, where it's used to compress the contents of QR codes (at least in cases where those QR codes contain large data, e.g. proofs). It takes up 46kb of space in the bundle:
<img width="413" alt="image" src="https://github.com/user-attachments/assets/e379b10e-d84a-42d2-8f29-ba74195bc3b7" />

Since it's already included by passport-ui, the additional cost of using it here is zero.